Formula/kstart.rb

```diff
@@ -3,6 +3,7 @@ class Kstart < Formula
   homepage "https://www.eyrie.org/~eagle/software/kstart/"
   url "https://archives.eyrie.org/software/kerberos/kstart-4.3.tar.xz"
   sha256 "7a3388ae79927c6698dc1bf20b29717e6bc34f692e00f12b3369d896f6702060"
+  license "MIT"
```

Comment:
That doesn't seem to match at all

Reply:
It is MIT license

```
# RPM spec file for kstart.
#
# Written by Russ Allbery <eagle@eyrie.org>
# Copyright 2015, 2021 Russ Allbery <eagle@eyrie.org>
# Copyright 2005-2012
#     The Board of Trustees of the Leland Stanford Junior University
#
# SPDX-License-Identifier: MIT

Name: kstart
Summary: Kerberos kinit variants supporting ticket refreshing
Version: 4.2
Release: 2%{?dist}
License: MIT
Group: System Environment/Base
URL: https://www.eyrie.org/~eagle/software/kstart/
Source: https://archives.eyrie.org/software/kerberos/%{name}-%{version}.tar.gz
BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root
BuildRequires: krb5-devel
Requires: krb5
Vendor: Stanford University
```

Reply:
I believe the Expat license corresponds to MIT. 
<details>
<summary>Here is snippet from README:</summary>

```
LICENSE

  The kstart package as a whole is covered by the following copyright
  statement and license:

    Copyright 2015, 2021 Russ Allbery <eagle@eyrie.org>
    Copyright 1995-1997, 1999-2002, 2004-2012, 2014
        The Board of Trustees of the Leland Stanford Junior University

    Permission is hereby granted, free of charge, to any person obtaining
    a copy of this software and associated documentation files (the
    "Software"), to deal in the Software without restriction, including
    without limitation the rights to use, copy, modify, merge, publish,
    distribute, sublicense, and/or sell copies of the Software, and to
    permit persons to whom the Software is furnished to do so, subject to
    the following conditions:

    The above copyright notice and this permission notice shall be
    included in all copies or substantial portions of the Software.

    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
    EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
    MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
    IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
    CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
    TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
    SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.

  Some files in this distribution are individually released under
  different licenses, all of which are compatible with the above general
  package license but which may require preservation of additional
  notices.  All required notices, and detailed information about the
  licensing of each file, are recorded in the LICENSE file.

  Files covered by a license with an assigned SPDX License Identifier
  include SPDX-License-Identifier tags to enable automated processing of
  license information.  See https://spdx.org/licenses/ for more
  information.

  For any copyright range specified by files in this package as YYYY-ZZZZ,
  the range specifies every single year in that closed interval.
```
</details>

There are some other licenses, but not sure if we need to account for them based on above comment that they are all compatible with general package license.

Reply:
In terms of SPDX which we use, I think they classify MIT and Expat as the same.

`kstart` does use SPDX identifiers in their code, e.g.
```console
❯ rg 'SPDX-License-Identifier: .*' --only-matching --no-filename --no-heading --no-line-number | sort | uniq -c | sort -r
  65 SPDX-License-Identifier: MIT
  22 SPDX-License-Identifier: FSFAP
  10 SPDX-License-Identifier: FSFULLR
   5 SPDX-License-Identifier: ISC
   2 SPDX-License-Identifier: \s-1FSFAP\s0
   1 SPDX-License-Identifier: \s+ \S+ }xms) {
```

The vast majority is `MIT`. There are some `FSFAP`, `FSFULLR`, and `ISC`, but some of these may not be used in Homebrew's formula and can be ignored (also licensing terms claim `MIT` should encompass these).

Reply:
Let's do it like this then:
```suggestion
  # technically the Expat license
  license "MIT"
```